### PR TITLE
Allow disable calico

### PR DESCRIPTION
**Pull request by jkoelker**
## High-level description

Reverts removal of option to disable calico overlay.

## Corresponding DC/OS tickets (required)

  - [COPS-6451](https://jira.d2iq.com/browse/COPS-6451) Should be able to disable Calico in config.yaml

### CODE REVIEW

**Comment by jkoelker** [comment]
@mesosphere-mergebot test  all

**Comment by jkoelker** [comment]
@mesosphere-mergebot test

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

**Comment by jkoelker** [comment]
@mesosphere-mergebot test all

**Comment by jkoelker** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by jkoelker** [comment]
@mesosphere-mergebot request-review @jongiddy @vinodkone

File: test-e2e/test_calico_networking.py [hunk 1]
```diff
@@ -37,6 +37,37 @@ def assert_system_unit_state(node: Node, unit_name: str, active: bool = True) ->
         assert "ConditionResult=no" in unit_properties
 
 
+def test_calico_disabled(docker_backend: Docker, artifact_path: Path,
```
Comment (jongiddy) [inline test-e2e/test_calico_networking.py:40]: Can we skip this test using `only_changed` unless the Calico package is modified?

**Review by jongiddy — APPROVED** [review]

File: test-e2e/test_calico_networking.py (hunk 1 above)
Comment (jkoelker) [reply test-e2e/test_calico_networking.py:40]: Yep I'll whip that up in a new pr quick since this one is in the train.

**Comment by jkoelker** [comment]
@mesosphere-mergebot backport  2.1

**Pull request merged**